cluster/gce/gci/configure-helper.sh

```diff
@@ -706,9 +705,6 @@ function start-kube-addons {
   fi
   if [[ "${ENABLE_L7_LOADBALANCING:-}" == "glbc" ]]; then
     setup-addon-manifests "addons" "cluster-loadbalancing/glbc"
```

Comment:
Sorry for late comment, I just came back to office. It is not a functionality problem, just an improvement. So, you don't have to make a PR dedicated for the correction. It is up to you